examples/src/main/java/io/dapr/examples/pubsub/http/Publisher.java

```diff
@@ -39,6 +39,9 @@ public class Publisher {
   //The title of the topic to be used for publishing
   private static final String TOPIC_NAME = "testingtopic";
 
+  // Topic registered with Bulk Subscribe handler.
+  private static final String BULK_SUB_TOPIC_NAME = "testingtopicbulk";

```

Comment:
We would still need different topics since there is only once instance of subscriber running with a single app-id. Have refactored publisher to use a topic from `args` and fallback to a default otherwise.